### PR TITLE
Refactor card description style

### PR DESCRIPTION
**Pull request by Cody-Price**
Refactor card description
footer
header
search bar with buttons

and conversion to scss

### CODE REVIEW

File: src/Search.scss [hunk 1]
```diff
@@ -6,59 +7,54 @@ input[type=search]{
...
-
-button:focus {
-  outline: none;
-}
 
-button:active {
-  background-color: #00ff00;
-  transition: 0s;
+  &:hover {
+    background-color: #00c7d6;
```
Comment (kayleenovak) [inline src/Search.scss]: Can this be changed to our blue-color variable?

File: src/Search.scss [hunk 2]
```diff
@@ -6,59 +7,54 @@ input[type=search]{
...
+    border-color: #00ff00;
```
Comment (kayleenovak) [inline src/Search.scss]: I think this is our color-green variable. Can this be changed too?

File: src/Search.scss [hunk 3]
```diff
@@ -6,59 +7,54 @@ input[type=search]{
...
+    transform: scale(1.15);
+    transition: .4s;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:active {
+    background-color: #00ff00;
```
Comment (kayleenovak) [inline src/Search.scss]: Same here - color-green

**Comment by Cody-Price** [comment]
I fixed all of the issues you pointed out, thank you kaylee!

**Comment by kayleenovak** [comment]
Great! Thank you for changing those! Looks great

**Pull request merged**